src/riscv_private.h

```diff
@@ -100,7 +102,7 @@ struct riscv_internal {
     riscv_word_t PC;
 
     /* user provided data */
-    riscv_user_t userdata;
+    riscv_user_t attr;
```

Comment:
Would it be better to name this field as `priv` or `data`?

Reply:
I think `data` is decent.